cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -122,6 +122,10 @@ available services which CM can install.
 +------------+-----------------------+--------------------------------------+
 | CM Version | CDH Version           | CDAP Parcel / CSD Version            |
 +============+=======================+======================================+
+| 5.9        | 5.9.x                 | |short-version-x|                    |
```

Comment:
we are missing the older CDH/CDAP versions for this (CM5.9), and CM 5.8 below.  We need to look up if there were any compatibility breaks between CDAP and CDH 5.7+ first